### PR TITLE
Improve robot_localiztion params

### PR DESCRIPTION
**Pull request by servos**
Moved robot_localization params into there own file and updated them based on experience with other robot platforms.

### CODE REVIEW

**Comment by servos** [comment]
@ayrton04 any comment on my r_l config for jackal?

File: jackal_control/config/robot_localization.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Q_vz: &Q_vz 1.0
+Q_vR: &Q_vR !degrees 90.0
+Q_vP: &Q_vP !degrees 90.0
+Q_vY: &Q_vY !degrees 90.0
+
+Q_ax: &Q_ax 1.0
+Q_ay: &Q_ay 1.0
+Q_az: &Q_az 1.0
+
+process_noise_covariance: [*Q_x, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
```
Comment (mikepurvis) [inline jackal_control/config/robot_localization.yaml]: Spiffy.

**Comment by servos** [comment]
I have removed the process covariance param changes so this PR now does 3 main things:
1) Adds the k_l and k_r params which are used in the cpr version of diff drive controller.
2) Removes differential pose inputs from the odom.
3) Move ekf params to there own file.

If we don't want to have the new file (I only did it because it is like that on other platforms) then I can revert that as well.

**Comment by ayrton04** [comment]
:+1:

**Comment by mikepurvis** [comment]
LGTM, thanks.

**Pull request merged**